lib/java/gradlew

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/env sh
```

Comment:
This script is autogenerated by the Gradle Wrapper integration. It is expected to exist and is updated as Gradle Wrapper is refreshed to a newer version.